lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -29,7 +29,7 @@ bool LLDBMemoryReader::queryDataLayout(DataLayoutQueryType type, void *inBuffer,
     // disk. Setting the bit in the mask ensures it isn't accidentally cleared
     // by ptrauth stripping.
     mask_pattern |= LLDB_FILE_ADDRESS_BIT;
-    memcpy(outBuffer, &mask_pattern, sizeof(uint64_t));
+    memcpy(outBuffer, &mask_pattern, m_process.GetAddressByteSize());
```

Comment:
The change itself is obviously good, but should we make sure that
```
  auto &triple = exe_module->GetArchitecture().GetTriple();
  if (triple.isArch64Bit()) {
```
also uses `m_process.GetAddressByteSize()` so the template paramter of the runtime and this condition is guaranteed to be in sync?